### PR TITLE
fix a component and add a WebSampleTest

### PR DESCRIPTION
**Pull request by kdenysiuk**
In the last commit I added the own xml (WebSampleTest) and running from IDEA it works, but I couldn't run it from maven, I searched from internet but still it doesn't work.

### CODE REVIEW

File: src/test/java/com/qaprosoft/carina/demo/WebSampleTest.java [hunk 1]
```diff
@@ -1,121 +1,43 @@
...
+		SignUpPage signUpPage = navigateMenu.clickSigninButton();
+		
+		//Create an account
+		signUpPage.fillUsernameField(R.TESTDATA.get("user_profile")+ new Random().nextInt());
+		signUpPage.fillPasswordField(R.TESTDATA.get("user_password"));
+		signUpPage.clickSigninButton();
+
+		//Assert of "Welcome" label
+		Assert.assertEquals(getDriver().switchTo().alert().getText(), "Sign up successful.", "Alert is wrong");
+		getDriver().switchTo().alert().accept();
```
Comment (ankochergin) [inline src/test/java/com/qaprosoft/carina/demo/WebSampleTest.java:37]: where is assert of "Welcome" label? you just checked the alert

File: src/test/resources/testng_suites/WebSampleTest.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
+<suite name="Web Sample Test">
+
```
Comment (ankochergin) [inline src/test/resources/testng_suites/WebSampleTest.xml]: please add parameters that are in example web.xml

File: src/test/java/com/qaprosoft/carina/demo/WebSampleTest.java [hunk 3]
```diff
@@ -1,121 +1,43 @@
...
-import com.qaprosoft.carina.demo.gui.pages.HomePage;
-import com.qaprosoft.carina.demo.gui.pages.ModelInfoPage;
-import com.qaprosoft.carina.demo.gui.pages.NewsPage;
+import com.qaprosoft.carina.demo.guipractice.components.NavigateMenu;
+import com.qaprosoft.carina.demo.guipractice.pages.HomePage;
+import com.qaprosoft.carina.demo.guipractice.pages.SignUpPage;
+
+import java.util.Random;
+
+public class WebSampleTest implements IAbstractTest{
```
Comment (ankochergin) [inline src/test/java/com/qaprosoft/carina/demo/WebSampleTest.java:15]: please add all you other tests

File: src/test/java/com/qaprosoft/carina/demo/WebSampleTest.java (hunk 1 above)
Comment (kdenysiuk) [reply src/test/java/com/qaprosoft/carina/demo/WebSampleTest.java:37]: Yes, I confused of method, the welcome label is according of the log in test, I will change the comment

File: src/test/java/com/qaprosoft/carina/demo/WebSampleTest.java [hunk 4]
```diff
@@ -32,12 +34,82 @@ public void testSignIn() {
...
+		Assert.assertEquals(getDriver().switchTo().alert().getText(), "Product added.", "Alert is wrong");
+		getDriver().switchTo().alert().accept();
+
+		//Go to cart
+		Assert.assertTrue(productPage.isPresentAddToCartButton(), "The \"Add To Cart\" is not present.");
+		navigateMenu.clickCartButton();
+		//Verify info
+
+//		Assert.assertEquals(.getProductTitle(), "Samsung galaxy s6", "Incorrect Specification");
+//		Assert.assertEquals(productPage.getProductPrice(), "$360", "Incorrect Specification");
```
Comment (ankochergin) [inline src/test/java/com/qaprosoft/carina/demo/WebSampleTest.java]: no verification after click cart

File: src/main/java/com/qaprosoft/carina/demo/guipractice/pages/CartPage.java [hunk 5]
```diff
@@ -26,4 +27,14 @@ public PurchasePage clickPlaceOrderButton() {
 		return new PurchasePage(driver);
 	}
 
+//	public ProductPage clickProduct(String productString) {
+//		for (Product product : products) {
+//			String currentProduct = product.getProductTitle();
+//			if ((productString).equalsIgnoreCase(currentProduct)) {
```
Comment (ankochergin) [inline src/main/java/com/qaprosoft/carina/demo/guipractice/pages/CartPage.java]: why it's commented?

**Comment by kdenysiuk** [comment]
Ready for Merging

**Pull request merged**